README.md

```diff
@@ -33,19 +30,26 @@ If you would like to update the current documentation, clone this repository, ma
 Copyright 2018 D. Meyer and M. Riechert.
 Released under [MIT License](LICENSE.txt).
 
-## How to cite
-Please refer to the [How to cite the GIS4WRF Plug-in page](https://doi.org/10.5281/zenodo.1288569).
+## How to reference GIS4WRF 
+We ask to please acknowledge our work by referencing GIS4WRF as follows --
+
+
+### In-text citations 
+
+Please cite both the paper and the software. E.g.
 
-## References
+*"We used GIS4WRF (Meyer and Riechert, 2018a, 2018b) for ..."*
+
+Their respective full citation can be found below under [Reference list](#reference-list) 
+
+### Reference list
+
+The full citation of GIS4WRF using the American Meterological Society (AMS) style can be found below. Please adapt using your chosen style.
 
 ```
-Meyer, D., and M. Riechert (2018a). GIS4WRF: an integrated open source QGIS toolkit for the Advanced 
+Meyer, D., and M. Riechert 2018a. GIS4WRF: an integrated open source QGIS toolkit for the Advanced 
     Research WRF Framework. Manuscript submitted for publication.
 
-Meyer, D., and M. Riechert (2018b). GIS4WRF (Version 0.8.0) [Software].
-    doi: 10.5281/zenodo.1288570.
-
-Skamarock, W. C., Klemp, J. B., Dudhia, J., Gill, D. O., Barker, D. M., Duda, M. G., Huang, X.-Y.,
-    Wang, W. and Powers, J. G. (2008) ‘A Description of the Advanced Research WRF Version 3’,
-    NCAR Technical Note NCAR/TN-475+STR, p. 113. doi: 10.5065/D68S4MVH.
+D. Meyer, and M. Riechert, 2018b: GIS4WRF. 
+    doi:10.5281/zenodo.1288570.
```

Comment:
This is the DOI for 0.8.0. We need to decide whether we use the "all versions" DOI or the version-specific variant. For the latter, we don't know the new DOI for 0.8.1 yet, and shouldn't we then include the version number in the title as we had it before?